website/docs/index.html.markdown

```diff
@@ -59,7 +59,7 @@ resource "azurestack_virtual_network" "test" {
 
 The following arguments are supported:
 
-* `arm_endpoint` - (Optional) The Azure Resource Manager Endpoint for your Azure Stack instance, for example `https://management.westus.mydomain.com`. This can also be sourceed from the `ARM_ENDPOINT` Environment Variable.
+* `metadata_hostname` - (Optional) The Azure Resource Manager FQDN for your Azure Stack instance, for example `management.westus.mydomain.com`. This can also be sourceed from the `ARM_METADATA_HOSTNAME` Environment Variable.
```

Comment:
```suggestion
* `metadata_host` - (Optional) The Azure Resource Manager FQDN for your Azure Stack instance, for example `management.westus.mydomain.com`. This can also be sourced from the `ARM_METADATA_HOSTNAME` Environment Variable.
```